### ISSUE TITLE
Loss function formula

### ISSUE THREAD

**Issue by Pichairen** [crisb-DUT/DnCNN-tensorflow#42]

**Comment by Pichairen** [comment]
Hi, thanks for share your code and it's helpful for me. I am puzzled about the loss function, as described in the paper,loss function as follow:
![image](https://user-images.githubusercontent.com/32973650/57970021-ac177680-79ae-11e9-8a2c-2ac1f254f0a6.png)
In this code, the loss is the MSE of  subtraction between noise-free image and predict residual map?
